tests/api/test_account.py

```diff
@@ -1131,6 +1132,47 @@ def test_staff_update(staff_api_client, permission_manage_staff, media_root):
     assert not data["user"]["isActive"]
 
 
+@patch("saleor.graphql.account.mutations.get_random_avatar")
```

Comment:
We could patch `create_user_avatar_thumbnails` and `assert mocked_thumbnail_creator.call_count == 0`